include/aws/common/windows/rw_lock.inl

```diff
@@ -18,9 +18,14 @@
 
 AWS_EXTERN_C_BEGIN
 
+#include <Windows.h>
```

Comment:
Same as with atomics, forward declare and get rid of this include

Reply:
Moved to .c file